### PR TITLE
created shareButton

### PR DESCRIPTION
**Pull request by Abdellah-Itrkine**

### CODE REVIEW

**Comment by soufyanelf** [comment]
Please fix the merge request try to pull the latest changes from **staging** branch

File: src/app/helps/[id]/ShareToSocialMediaButton/index.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { ThumbUpAltSharp, ThumbUpOffAlt } from "@mui/icons-material";
+import { Button, CircularProgress, Typography } from "@mui/material";
+
+export const ShareButton = () => {
+
+
+
+
+  // Define the URL to share (replace with your URL)
+  const shareUrl = "https://example.com"; // Replace with your URL
```
Comment (soufyanelf) [inline src/app/helps/[id]/ShareToSocialMediaButton/index.jsx:15]: add the right URL

File: src/app/helps/[id]/page.jsx [hunk 2]
```diff
@@ -41,6 +42,16 @@ async function getHelp(helpId) {
   }
 }
 
+
+
+const urlToShare = 'https://example.com'; // Replace with your URL
```
Comment (soufyanelf) [inline src/app/helps/[id]/page.jsx:47]: we already have the shareUrl inside the component

File: src/app/helps/[id]/page.jsx [hunk 3]
```diff
@@ -41,6 +42,16 @@ async function getHelp(helpId) {
...
+
+
```
Comment (soufyanelf) [inline src/app/helps/[id]/page.jsx:49]: remove empty lines

File: src/app/helps/[id]/ShareToSocialMediaButton/index.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+"use client";
+import React from 'react';
+
+import { useConfirmation } from "@/hooks/useConfimration";
+import { useDisConfirmation } from "@/hooks/useDisConfirmation";
+import { ThumbUpAltSharp, ThumbUpOffAlt } from "@mui/icons-material";
+import { Button, CircularProgress, Typography } from "@mui/material";
```
Comment (soufyanelf) [inline src/app/helps/[id]/ShareToSocialMediaButton/index.jsx:7]: why are you importing those?

File: package.json [hunk 5]
```diff
@@ -17,9 +17,9 @@
     "eslint": "8.49.0",
     "eslint-config-next": "13.4.19",
     "firebase": "^10.3.1",
-    "next": "13.4.19",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "next": "^13.5.2",
```
[the remark this answers is not in the document]
Comment (Abdellah-Itrkine) [reply package.json:20]: about this i don't know why older version did't work with me on my machine

File: src/app/helps/[id]/ShareToSocialMediaButton/index.jsx (hunk 1 above)
Comment (Abdellah-Itrkine) [reply src/app/helps/[id]/ShareToSocialMediaButton/index.jsx]: done

File: src/app/helps/[id]/page.jsx (hunk 2 above)
Comment (Abdellah-Itrkine) [reply src/app/helps/[id]/page.jsx]: i did't notice it's now a paramter

File: src/app/helps/[id]/page.jsx (hunk 3 above)
Comment (Abdellah-Itrkine) [reply src/app/helps/[id]/page.jsx]: removed

File: src/app/helps/[id]/ShareToSocialMediaButton/index.jsx (hunk 4 above)
Comment (Abdellah-Itrkine) [reply src/app/helps/[id]/ShareToSocialMediaButton/index.jsx]: i made a copy and i did not notice that i keeped them

**Comment by Abdellah-Itrkine** [comment]
Check this plz and let me know what should be changed

**Comment by soufyanelf** [comment]
@Abdellah-Itrkine  you didn't respect the mockup?
image

**Comment by soufyanelf** [comment]
@Abdellah-Itrkine  please respect the design if you're not available I'll jump to work on those issues

**Pull request closed without merging**